.env.example

```
@@ -24,20 +24,17 @@ OPENAI_BASE_URL=https://api.openai.com/v1
 RESEND_API_KEY=re_...
 RESEND_FROM=Example <noreply@example.com>
 
-# AWS S3 / compatible storage
+# AWS S3 / compatible storage (private bucket — question batches, etc.)
 AWS_ACCESS_KEY_ID=...
 AWS_SECRET_ACCESS_KEY=...
 # Optional: temporary / assumed-role credentials
 # AWS_SESSION_TOKEN=...
-AWS_REGION=us-east-1
-AWS_S3_BUCKET=your_bucket_name
+AWS_REGION=auto
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AWS_REGION key should go before the AWS_SECRET_ACCESS_KEY key

(UnorderedKey)